### PR TITLE
Run tests when PR is merged into release-0.3 branch

### PR DESCRIPTION
**Pull request by msajidmansoori12**
This PR enables running tier3 tests for commits/PRs merged into release-0.3 branch.

### CODE REVIEW

**Review by aufi — COMMENTED** [review]
Thanks for the PR. I'm not sure how this workflow is triggered, since there doesn't seem to be after merge hook for this task, it looks to me to be triggered just by cron on an internal machine, am I wrong?

File: README.md [hunk 1]
```diff
@@ -5,11 +5,12 @@
 [![Test TIER0](https://github.com/konveyor/go-konveyor-tests/actions/workflows/main-tier0.yml/badge.svg)](https://github.com/konveyor/go-konveyor-tests/actions/workflows/main-tier0.yml)
 [![Test TIER1](https://github.com/konveyor/go-konveyor-tests/actions/workflows/main-tier1.yml/badge.svg)](https://github.com/konveyor/go-konveyor-tests/actions/workflows/main-tier1.yml)
 [![Test TIER2](https://github.com/konveyor/go-konveyor-tests/actions/workflows/main-tier2.yml/badge.svg)](https://github.com/konveyor/go-konveyor-tests/actions/workflows/main-tier2.yml)
-[![TEST TIER3](https://img.shields.io/endpoint?url=https%3A%2F%2Fsajidmansoori12.pythonanywhere.com%2Fretrieve_data&cacheSeconds=60)](https://main-jenkins-csb-migrationqe.apps.ocp-c1.prod.psi.redhat.com/view/MTA/job/mta/job/konveyor-go-tests-pr-tester/job/main/)
+[![TEST TIER3](https://img.shields.io/endpoint?url=https%3A%2F%2Fsajidmansoori12.pythonanywhere.com%2Fretrieve_data%3Fpipeline%3Dtier3-pr&cacheSeconds=60)](https://main-jenkins-csb-migrationqe.apps.ocp-c1.prod.psi.redhat.com/view/MTA/job/mta/job/konveyor-go-tests-pr-tester/job/main/)
```
[the remark this answers is not in the document]
Comment (msajidmansoori12) [reply README.md:8]: @aufi this should be okay I think, we are doing the same on main branch as well. This url will only be accessible if the user is connected to RH VPN.

**Comment by msajidmansoori12** [comment]
> Thanks for the PR. I'm not sure how this workflow is triggered, since there doesn't seem to be after merge hook for this task, it looks to me to be triggered just by cron on an internal machine, am I wrong?

@aufi You are correct, we have a job which checks for new branches/PRs (cron based) since it's not possible to trigger the job via a webhook because our jenkins is hosted behind the VPN.

**Review by aufi — APPROVED** [review]
Thanks for exaplanation @msajidmansoori12

**Comment by mguetta1** [comment]
Thanks @msajidmansoori12!
We need to understand why the PR tester job fails for this PR before merging.

**Comment by mguetta1** [comment]
Changes have been made in another PR

**Pull request closed without merging**